src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -66,7 +67,12 @@ module.exports = class ArtworkCommercialView extends Backbone.View
         user: loggedInUser
       .then (data) ->
         { order, error } = data?.ecommerceCreateOrderWithArtwork?.orderOrError?
```

Comment:
I had fixed this in [this PR](https://github.com/artsy/force/pull/2955/files#diff-bee1c12adabe31f9b7c42e24d59e71b7R68), but make sure to get rid of the trailing `?` so you actually return the objects on this line.

Reply:
Good catch. I thought destructuring would work fine in this context, but it may not. I'll update it.